Tests/PackageLoading/ConventionTests.swift

```diff
@@ -277,11 +276,10 @@ final class PackageBuilderTester {
             }
         }
 
-        func checkSources(root: String) {
-            XCTAssertEqual(module.sources.root, AbsolutePath(root))
-        }
-
-        func checkSources(_ paths: String...) {
+        func checkSources(root: String? = nil, paths: String...) {
+            if let root = root {
+                XCTAssertEqual(module.sources.root, AbsolutePath(root))
+            }
             for path in paths {
                 XCTAssert(sources.contains(RelativePath(path)), "\(path) not found in module \(module.name)")
             }
```

Comment:
Yeah, I think so. We can have an init that takes a `[String]` if there are tests that need to factor out the list construction across separate lines.